src/Discovery/DiscoveryDetails/DataDownloadList/DataDownloadList.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Collapse, List, Button } from 'antd';
+import { DownloadOutlined } from '@ant-design/icons';
+import CheckThatDataHasTitles from './Utils/CheckThatDataHasTitles';
+import DataDownloadListItem from './Utils/DataDownloadListItem';
+
+const { Panel } = Collapse;
+
+const DataDownloadList = ({ sourceFieldData }: any) => {
+  const data = sourceFieldData[0].map((obj) => (
+    {
+      title: obj.title,
+      description: obj.description,
+    }
+  ),
+  );
+  if (CheckThatDataHasTitles(data) === false) return null;
+  return (
+    <Collapse defaultActiveKey={['1']}>
```

Comment:
I know this was implenented using `Collapse` in the non-tabbed version of details page and we are porting it over straightly, but I think for this effort we can unwrap it and uses `List` directly

Reply:
Removed the Collapse and Panel, now using List directly.